src/main/scala/com/databricks/spark/redshift/Parameters.scala

```diff
@@ -192,7 +192,14 @@ private [redshift] object Parameters extends Logging {
           }
         }
 
-      s"aws_access_key_id=$accessKeyId;aws_secret_access_key=$secretAccessKey"
+      val credentials = s"aws_access_key_id=$accessKeyId;aws_secret_access_key=$secretAccessKey"
+
+      if(parameters.contains("aws_security_token")) {
```

Comment:
Style nit: please add a space after the `if` token (I just implemented automated Scalastyle checks, which would have caught this).